app/src/store/StoreAlerts.re

```diff
@@ -0,0 +1,30 @@
+open Molecule.Alert;
+
+type alert = {
+  severity,
+  message: string,
+  id: int,
+};
+
+type action =
+  | Show(alert)
+  | Remove(alert);
```

Comment:
This only depends on the `id` not the full alert, would make the surface smaller.

Reply:
Ha, was thinking of this before, but wanted the alert system to be 1000% robust, you know, in case the random ids clash ; P

Will try it out!

Reply:
I like that the interface is the same for all the actions. I can change it to reduce the surface area if you insist.

Reply:
I don't. :> Smaller interfaces just make it easier to reason about and the added benefit is that the caller side needs to hold on to less data, needs less to fullfill the call.